locust/argument_parser.py

```diff
@@ -23,7 +23,7 @@
 version = locust.__version__
 
 
-DEFAULT_CONFIG_FILES = ["~/.locust.conf", "locust.conf"]
+DEFAULT_CONFIG_FILES = ("pyproject.toml", "~/.locust.conf", "locust.conf")
```

Comment:
I think the order here matters, so put pyproject.toml last, so they are same order as they are documented :)

Reply:
… or change the .rst..

Reply:
The order is important, but it functions in the opposite way; the [configargparse](https://github.com/bw2/ConfigArgParse) reverses the list of configuration files internally.
<img width="1242" alt="Screenshot 2024-02-25 at 10 56 08" src="https://github.com/locustio/locust/assets/87263895/de9fb9c2-ca14-496c-9fdc-47d55e913950">

The DEFAULT_CONFIG_FILES are in the correct order, and I have defined the same in the updated documents.
`./pyproject.toml -> ~/.locust.conf -> ./locust.conf -> (file specified using --conf) -> env vars -> cmd args`.

Reply:
I made another comment in the place where it was confusing. 

Anyways the global file should be overridden by local files so I'd like it to be ~/.locust.conf, ./locust.conf, ./pyproject.toml